### ISSUE TITLE
Any plan to support advfirewall?

### ISSUE THREAD

**Issue by chucklu** [rpetz/SharpNetSH#13]
The last commit in this repository is  created on Jul 13, 2017.
Is this repository status in dormant?

**Comment by rpetz** [comment]
Yes, I have not had any time to put into this project in several years - it is still pretty straightforward if you would like to add to it, but it definitely is not complete by any means
